[New] Add raster from service (#425)
Esri/arcgis-maps-sdk-swift-samples: Shared/Samples/Add raster from service/README.md

# Add raster from service

Create a raster layer from a raster image service.

> ![Image of add raster from service](add-raster-from-service.png)

Reviewer: ```suggestion
![Image of Add raster from service sample](add-raster-from-service.png)
```

## Use case

Accessing a raster image from an online service can be useful for analysing the most up-to-date data available for an area. For example, retrieving recent results of bathymetry surveys within a shipping channel monitored for its sediment build-up would allow planners to assess dredging needs.

## How to use the sample

Simply launch the sample to see a raster from an image service being used on a map.

## How it works

1. Create an `ImageServiceRaster` using the service's URL.
2. Create a `RasterLayer` from the image service raster.

> 3. Add the raster layer the the map.

Reviewer: ```suggestion
3. Add the raster layer to the map.
```

## Relevant API

* ImageServiceRaster
* RasterLayer

## About the data

> This sample uses a [NOAA raster image service](https://gis.ngdc.noaa.gov/arcgis/rest/services/bag_hillshades_subsets/ImageServer). The service computes a hillshade image from the depth (in meters) of U.S. coastal waters.

Reviewer: ```suggestion
This sample uses a [NOAA raster image service](https://gis.ngdc.noaa.gov/arcgis/rest/services/bag_bathymetry/ImageServer). The service computes a hillshade image from the depth (in meters) of U.S. coastal waters.
```
